mbway/src/main/java/com/adyen/checkout/mbway/internal/ui/MBWayComponent.kt

```diff
@@ -77,8 +80,11 @@ internal class MBWayComponent(
             .stateIn(coroutineScope, SharingStarted.Lazily, stateManager.state.value.toViewState())
     }
 
+    private var isLoading by mutableStateOf(false)
```

Comment:
I think we can include this in the `ViewState` ( we probably will need it to the internal `ComponentState` as well since view state is generated based on that), then get it from there to pass it to the `PayButton`.

Reply:
Nice idea, let me try it out